src/htable.cpp

```diff
@@ -924,24 +925,24 @@ HeadedTable::HeadedTable(QWidget *parent, int opts) : QWidget(parent)
     // synchronize horizontal scrolling of head and body
     connect(body->horizontalScrollBar(), SIGNAL(valueChanged(int)), head,
             SLOT(scrollSideways(int)));
-    fontChange(font()); // *** need for init
+    fontChange(); // *** need for init
```

Comment:
It seems that with your change (which is good), `fontChange` isn't needed here(?) It didn't set the cell height correctly either.